python/llm/test/benchmark/whisper/README.md

```diff
@@ -0,0 +1,35 @@
+# Whisper Test
+The Whisper Test allows users to evaluate the performance and accuracy of Whisper speech-to-text models.
+For accuracy, the model is tested on the [LibriSpeech] (https://huggingface.co/datasets/librispeech_asr) dataset using [Word Error Rate (WER)](https://github.com/huggingface/evaluate/tree/main/metrics/wer) metric.
```

Comment:
"[LibriSpeech] (https://huggingface.co/datasets/librispeech_asr)" there is a redundant space.